src/Platform/ColumnType.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\SynapseTransformation\Platform;
+
+use Keboola\Datatype\Definition\BaseType;
+use Keboola\Datatype\Definition\Common;
+use Keboola\SynapseTransformation\Exception\NotImplementedException;
+use Keboola\SynapseTransformation\Exception\UnexpectedColumnType;
+
+class ColumnType extends Common
```

Comment:
nepatri tohle spis do php-datatypes?

Reply:
patri, ale tam by som musel na to spravit uplnu podporu nie? co zatial netreba (?)

Reply:
ak to moze ist v tomto stave do `php-datatypes` tak to presuniem

Reply:
hele nevim - to je asi spis na domluve s connection - tam to bude afaik taky potreba, tak to asi zajca? bude nejlip vedet. V kazdym pripade, pokud by to melo zustat tady, tak by melo byt zalozeny issue v jire a komentar v kodu s odkazem, aby to bylo jasny i za rok, ze to sem nepatri

Reply:
@zajca , chces aby som tento subor: https://github.com/keboola/synapse-transformation/blob/a2716b7872bf3ce9cdea20e6a71342c2486335f3/src/Platform/ColumnType.php .... presunul do `php-datatypes` aj ked tam je implementovany iba `type-mapping` a ostatne veci nie? ... alebo to necham tu a vytvorim issue na Jire a do kodu dam link na issue ...?

Reply:
php-datatypes už synapse má: https://github.com/keboola/php-datatypes/pull/26
kdyžtak daj do composeru zatím `dev-zajca-kbc-166-synapse`
Když to bude OK, tak uděláme merge, zatím sme to nedali do masteru protože sme to neměli jak otestovat v reálu.

Reply:
vdaka Zajca.

ok. upravene v https://github.com/keboola/synapse-transformation/pull/1/commits/5b9f9ca3745c418670b07b73628c4bba35cbed4b

Boli tam 2 issues:
- ja som mal typ NUMERIC aj tam kde malo byt FLOAT
- db reportuje `length` aj pre typy, ktore ju maju staticku (napr. `bigint` 8 bajtov), ... fixol som to  v kode aj v testoch